package.json

```diff
@@ -46,6 +48,7 @@
     "@types/chai": "^4.2.11",
     "@types/jsdom": "^16.2.3",
     "@types/mocha": "^9.1.0",
+    "@types/node": "^17.0.44",
```

Comment:
TODO: I think we can back this change out as this was from earlier iteration which attempted to use `fs` and `path`.